_data/tools_list.yml

```diff
@@ -0,0 +1,228 @@
+- name: Crate-O
+  description: Browser-based editor for Research Object Crates, inspired by Describo, with vocabulary support for Schema.org Style schemas (SOSSs)
+  status: alpha
+  level: 
```

Comment:
We intended this to be populated with "beginner", "intermediate", "advanced" so that users would have a sense of how much expertise they needed in order to use the tool. I do think this would be helpful as there is a wide range here, some tools you can use without much experience with programming and/or RO Crate, but others require advanced knowledge. I left it out of the table for now because we'll need to go through and populate the values.

Reply:
Ah, got it. Good idea to add that in future